pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -404,6 +404,16 @@ func (oe openshiftExtension) SetAuthBaseURL(ctx context.Context, th *v1alpha1.Te
 			return err
 		}
 		th.Status.SetAuthRoute(fmt.Sprintf("https://%s", authRoute))
+
+		if string(secret.Data["AUTH_BASE_URL"]) != th.Status.AuthRouteUrl {
+			secret.StringData = make(map[string]string)
```

Comment:
why are we initializing it, we should do it if it is nill not every time

also if we are initializing it now, then if will give Nil Pinter Exception I think

Reply:
have updated the patch,please take a look at it once